app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/EventShowcaseActivity.java

```diff
@@ -131,9 +132,15 @@ protected void onCreate(Bundle savedInstanceState) {
                 this.setupMenu(ev);
             });
 
+            if(getIntent().getIntExtra(getString(R.string.tag_facebook_login_back_button), -1) == TAG_NUMBER_FACEBOOK) {
```

Comment:
You could remove this method if only the other suppressive check is used and this one does not trigger any action.